### DISCUSSION TITLE
WindowsComponent- Background color change

### DISCUSSION THREAD

**Discussion by pokkireddy** [TouK/nussknacker#4075] [Q&A]
How to change the background color for the all windows component popup?

I have managed to change the color for the the footer ./windowmanager/WindowContent.tsx but unable to find to change for the entire component 
![Windows-Component](https://user-images.githubusercontent.com/63649118/223573100-8a769e5f-d879-4656-8930-18145b7936ac.JPG)

**Comment by jzdr** [comment]
There is no mechanizm to change that colour. If you really want to do it, unfortunately you have to find all the places in the code to change. You should probably start with these files:
designer/client/containers/darkTheme.tsx
designer/client/stylesheets/darkColors.styl
designer/client/stylesheets/_variables.styl
